base/math.jl

```diff
@@ -360,6 +360,14 @@ asinh(x::Number)
     expm1(x)
 
 Accurately compute ``e^x-1``.
+# Examples
+```jldoctest
+julia> expm1(2)
```

Comment:
Would comparing `expm1(1e-16)` with `exp(1e-16) - 1` be a better example here? Otherwise I could see new users being confused why this exists at all.

Reply:
Yes, thats a nice idea I will do that